src/widgets/Window.cpp

```diff
@@ -660,6 +660,53 @@ void Window::addShortcuts()
              }
              return "";
          }},
+        {"setShowLiveOnly",
+         [this](std::vector<QString> arguments) -> QString {
+             auto mode = 2;
+             if (arguments.size() != 0)
+             {
+                 auto arg = arguments.at(0);
+                 if (arg == "off")
+                 {
+                     mode = 0;
+                 }
+                 else if (arg == "on")
+                 {
+                     mode = 1;
+                 }
+                 else if (arg == "toggle")
+                 {
+                     mode = 2;
+                 }
+                 else
+                 {
+                     qCWarning(chatterinoHotkeys)
+                         << "Invalid argument for setShowLiveOnly hotkey: "
+                         << arg;
+                     return QString("Invalid argument for setShowLiveOnly "
+                                    "hotkey: %1. Use \"on\" or \"off\".")
+                         .arg(arg);
+                 }
+             }
+
+             if (mode == 0)
+             {
+                 getSettings()->showLiveOnly.setValue(false);
+                 this->notebook_->setShowLiveOnly(false);
+             }
+             else if (mode == 1)
+             {
+                 getSettings()->showLiveOnly.setValue(true);
+                 this->notebook_->setShowLiveOnly(true);
+             }
+             else if (mode == 2)
+             {
+                 auto value = this->notebook_->getShowLiveOnly();
+                 getSettings()->showLiveOnly.setValue(!value);
+                 this->notebook_->setShowLiveOnly(!value);
+             }
+             return "";
```

Comment:
You've essentially duplicated your if/else statements here. I'd recommend that you move all of your `mode` conditional logic into the first lot of statements and remove the `mode` variable altogether.

Reply:
Thanks for the review! I simplified this bit in 51ac47b.